README.md

```diff
@@ -37,6 +37,8 @@ or use directly from unpkg:
 First, you'll want to initialize StimulusJS and then you can import all the TailwindCSS components.
 
 ```javascript
+// app/javascript/controllers/index.js
```

Comment:
This path can vary depending on the version of webpacker: 

https://github.com/rails/webpacker/blob/master/6_0_upgrade.md#preparation

```suggestion
// webpacker 5: app/javascript/controllers/index.js
// webpacker 6: app/packs/controllers/index.js
```